src/map/entities/charentity.cpp

```diff
@@ -234,6 +234,8 @@ CCharEntity::CCharEntity()
 
     m_Substate = CHAR_SUBSTATE::SUBSTATE_NONE;
 
+    m_LeaderCreatedPartyTime = server_clock::now() - std::chrono::seconds(120); // default time
```

Comment:
Why is this being set on character creation? (log in and zoning)

Reply:
mistake on my part, thought we needed it, removed now